src/scalper_utils.ts

```diff
@@ -307,4 +310,67 @@ export async function getFairValue(connection, spotMarket, symbol) {
     }
   }
   return fairValue;
+}
+
+export async function jupiterHedge(connection: Connection, owner: Keypair, hedgeSide: string, 
+    base: string, quote: string, hedgeDelta: number, hedgePrice: number) {
+  const jupiter = await Jupiter.load({
+    connection: connection,
+    cluster: cluster,
+    user: owner, 
+    wrapUnwrapSOL: false, 
+  });
+  let inputToken: PublicKey;
+  let outputToken: PublicKey;
+  let hedgeAmount: number;
+  if (hedgeSide == "sell"){
+    inputToken = new PublicKey(tokenToSplMint(base));
+    outputToken = new PublicKey(tokenToSplMint(quote));
+    hedgeAmount = Math.abs(hedgeDelta);
+  } else {
+    inputToken = new PublicKey(tokenToSplMint(quote));
+    outputToken = new PublicKey(tokenToSplMint(base));
+    hedgeAmount = Math.abs(hedgeDelta) * hedgePrice;
+  }
+  const inputQty = Math.round(hedgeAmount * (10 ** decimalsBaseSPL(splMintToToken(inputToken)))) // Amount to send to Jupiter
+  // Find best route, qty & price
+  let arbQty: number;
+  let arbValue:[number, number, string, {setupTransaction?: Transaction, swapTransaction: Transaction, cleanupTransaction?: Transaction}];
```

Comment:
best to make a named type for this so we can have better type checking in the usage and it will be a lot clearer when using it in the caller

Reply:
added RouteDetails